Documentation/contributor-guide/community-membership.md

```diff
@@ -103,39 +103,41 @@ depend on to make decisions in the best interest of the project in a consistent
 
 **Defined by:** *approvers* entry in the [OWNERS] file.
 
-### Requirements
+### Maintainer requirements
 
 - Deep understanding of the technical goals and direction of the project
 - Deep understanding of the technical domain of the project
 - Sustained contributions to design and direction by doing all of:
-    - Authoring and reviewing proposals
-    - Initiating, contributing, and resolving discussions (emails, GitHub issues, meetings)
-    - Identifying subtle or complex issues in the designs and implementation of PRs
+  - Authoring and reviewing proposals
+  - Initiating, contributing, and resolving discussions (emails, GitHub issues, meetings)
+  - Identifying subtle or complex issues in the designs and implementation of PRs
 - Directly contributed to the project through implementation and/or review
 - Sponsored by two active maintainers and elected by supermajority
-    - Sponsors must be from multiple member companies to demonstrate integration across the community.
-- To become a maintainer send an email with your candidacy to etcd-maintainers-private@googlegroups.com
-    - Ensure your sponsors are @mentioned in the email
-    - Include a list of contributions representative of your work on the project.
-    - Existing maintainers vote will privately and respond to the email with either acceptance or feedback for suggested improvement.
+  - Sponsors must be from multiple member companies to demonstrate integration across the community.
+- To become a maintainer send an email with your candidacy to <etcd-maintainers-private@googlegroups.com>
+  - Ensure your sponsors are @mentioned in the email
+  - Include a list of contributions representative of your work on the project.
+  - Existing maintainers vote will privately and respond to the email with either acceptance or feedback for suggested improvement.
 - With your membership approved you are expected to:
   - Open a PR and add an entry to the [OWNERS] file
-  - Subscribe to etcd-maintainers@googlegroups.com and etcd-maintainers-private@googlegroups.com
-  - Request to join [etcd-maintainer teams of etcd organization of GitHub](https://github.com/orgs/etcd-io/teams/maintainers-etcd)
+  - Subscribe to <etcd-maintainers@googlegroups.com> and <etcd-maintainers-private@googlegroups.com>
```

Comment:
I tried subscribing to this mailing list, but I got:

> The group you attempted to subscribe to is invite only. You must be invited by the group owner.

Would this need to be changed to request an invitation to the mailing group from [Someone]?